src/components/AdminSettings.vue

```diff
@@ -286,9 +450,28 @@ export default {
 		isOPOAuthFormInDisableMode() {
 			return this.formMode.opOauth === F_MODES.DISABLE
 		},
+		isOPUserAppPasswordFormInEdit() {
+			return this.formMode.opUserAppPassword === F_MODES.EDIT
+		},
+		isGroupFolderSetupFormInEdit() {
+			return this.formMode.groupFolderSetUp === F_MODES.EDIT
+		},
 		isNcOAuthFormInDisableMode() {
 			return this.formMode.ncOauth === F_MODES.DISABLE
 		},
+		isGroupFolderSetUpInDisableMode() {
+			return this.formMode.groupFolderSetUp === F_MODES.DISABLE
+		},
+		isOPUserAppPasswordInDisableMode() {
+			return this.formMode.opUserAppPassword === F_MODES.DISABLE
+		},
+		isGroupFolderSetupCorrectAfterSetup() {
+			return (this.opUserAppPassword && this.formMode.groupFolderSetUp !== F_MODES.EDIT && this.isGroupFolderSetupCorrect === false)
```

Comment:
Shouldn't this be true?
```suggestion
			return (this.opUserAppPassword && this.formMode.groupFolderSetUp !== F_MODES.EDIT && this.isGroupFolderSetupCorrect === true)
```